src/lib/tab_groups.ts

```diff
@@ -0,0 +1,330 @@
+import { browserBg, activeTabId } from "./webext"
+
+/**
+ * Return the active window's id.
+ *
+ */
+export async function activeWindowId() {
+    return (await browserBg.windows.getCurrent()).id
+}
```

Comment:
I think it might be better to put this function near activeTabId, since they're similar.